website/materials.html

```diff
@@ -0,0 +1,111 @@
+  <head>
+    <title>CHART</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+<!--my css code. -->
+    <link rel="stylesheet" href="index.css" type="text/css">
+
+  </head>
+<!--I am also having a hard time spreading the nav bar equally spaced (maybe use an id)
+I also can't get the drop down menu to work-->
+<header id="mainmenu">
+  <nav class="navbar navbar-expand-lg nav-justified" style="background:black;">
+    <div>
+  <a class="navbar-brand text-white" href="../index.html"> </a> </div>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse justify-content-center nav-fill" id="navbarSupportedContent">
+    <ul class="nav navbar-nav nav-fill" class= ".active">
+      <li class="active">
+        <a class="nav-link text-white" href="../index.html">Home<span class="sr-only">(current)</span></a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-white" href="about.html">About</a></li>
+      <li class="nav-item">
+        <a class="nav-link text-white" href="materials.html">Materials</a> </li>        <li class="nav-item dropdown">
+       <a class="nav-link dropdown-toggle text-white" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Tutorials
+          </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item text-black" href="hardware.html">Telescope Design</a>
+            <a class="dropdown-item text-black" href="software.html">Software Setup</a>
+            <a class="dropdown-item text-black" href="datataking.html">Taking data</a>
+            <a class="dropdown-item text-black" href="analysis.html">Analysis</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="faq.html">FAQ</a> </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="contact.html">Contact</a> </li>
+        </ul>
+</div>
+</div>
+</nav>
+<h2 id="centerheader" style= "border:solid;border-color:black;" font-family:"courier new", courier, arial; >Suggested Materials</h2>
+</header>
+
+
+
+  <div class="container-fixed" style="padding:0px; margin:0px;">
+  <section class="section">
+  <p style="margin:20px;"> This list is intended to serve as a starting place for some suggested materials to build a version of CHART.
+    
+    <br> Materials for building:
```

Comment:
This line doesn't show up on my screen, it's under the heading:
![image](https://user-images.githubusercontent.com/4752917/90647044-beedac00-e1ec-11ea-8e49-e5467060dc26.png)

Also, maybe add a few more words to describe _what_ you're building with these things? Or categorize a little, with some formatting to make it easier to look at.